.travis.yml

```diff
@@ -1,12 +1,13 @@
-language: swift
+language: objective-c
```

Comment:
What was the reason behind this change? I think we should keep this as `swift` since the project is primarily written in Swift.

Reply:
That’s the way it’s always been,
following syntax. Per Travis docs:

> Right now, language: swift is just an alias for language: objective-c. Said another way, we don’t have native support for Swift projects at this time. Swift builds are just routed to our macOS image, the same as Objective-C builds.

https://docs.travis-ci.com/user/languages/objective-c/

The Realm build doesn’t succeed in Xcode so it’s not just the CI VM

Reply:
Even if they're an alias now we want to pick Swift since that's what the vast majority of our code is written in. If Travis ever changes to add Swift specific support in the future we'd want to take advantage of that.

Reply:
sure thing. I've got it running on the realm-downgrade branch. Thanks for the review